tests/external_botocore/test_bedrock_embeddings.py

```diff
@@ -19,8 +19,8 @@
 import pytest
 from _test_bedrock_embeddings import (
     embedding_expected_client_errors,
-    embedding_expected_events,
     embedding_expected_error_events,
```

Comment:
Same thing here-basically a no-op. Just drop it from the list of changes.

Reply:
I figure I might keep this one only because this one is a linter formatting thing where it takes the imports and alphabetizes them.